double/s_scalbn.c

```diff
@@ -63,6 +63,6 @@ gm_scalbn (double x, int n)
         return x*twom54;
 }
 
-#if defined(__clang__) || defined(__GNUC__) 
+#if (defined(__clang__) || defined(__GNUC__)) && !defined(__APPLE__)
```

Comment:
Does the apple build really not support this syntax? All the version of clang I can test on godbolt.org accept this.

Reply:
float/s_scalbnf.c:54:54: error: aliases are not supported on darwin
extern __typeof (gm_scalbnf)gm_ldexpf __attribute__((__alias__("gm_scalbnf")));

Gives me this, so Im afraid is not supported on this platform